cmd/odf/set/log_level.go

```diff
@@ -11,11 +11,11 @@ import (
 
 var setLogLevelCmd = &cobra.Command{
 	Use:                "log-level",
-	Short:              "Set different log levels for ceph dameons like mon, osd and mds",
-	Example:            "odf set log-level osd 20",
+	Short:              "Set different log levels for ceph components like mon, osd and mds",
+	Example:            "odf set log-level <ceph_component> <ceph_subsystem> <{log-level}/{memory-level}>",
```

Comment:
Let's just keep this as an example. The docs can give the details about the params.
```suggestion
	Example:            "odf set log-level osd crush 10",
```